net/browserless_agent/lib/browserless_agent.js

```diff
@@ -0,0 +1,201 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.browserlessAgent = void 0;
+const browserlessAgent = async ({ namedInputs, params }) => {
+    const { url, operation, options, headers, cookies } = namedInputs;
+    const throwError = params?.throwError ?? false;
+    const browserlessEndpoint = "https://chrome.browserless.io";
+    const browserlessToken = process.env.BROWSERLESS_API_TOKEN;
+    // Check if API token is provided
+    if (!browserlessToken) {
+        const errorMessage = "Browserless API token is required. Please set the BROWSERLESS_API_TOKEN environment variable.";
+        throw new Error(errorMessage);
+    }
+    // Build the endpoint with token
+    const endpoint = `${browserlessEndpoint}?token=${browserlessToken}`;
+    // Select appropriate endpoint based on operation type
+    const operationType = operation || "content";
+    let apiPath;
```

Comment:
We write immutable code. Please update this to const using either a ternary expression or an immediately invoked function.